### DISCUSSION TITLE
JPEGViewLinear

### DISCUSSION THREAD

**Discussion by KrokusPokus** [sylikc/jpegview#334] [General]
@sylikc 
For what it's worth, I made that private fork of JPEGView with SSE2 accelerated linear light resampling available at https://github.com/KrokusPokus/JPEGViewLinear.
Note that this is a pure viewer that I originally made for getting better image quality while reading manga. It has little in common with regular JPEGView.

**Comment by haico1992** [comment]
Thanks!

**Comment by KrokusPokus** [comment]
Well, I finally ended up updating the old fork. It now is based on the the current sylikc version and can be found here:
https://github.com/KrokusPokus/JPEGView_L
